### DISCUSSION TITLE
osu!Quests

### DISCUSSION THREAD

**Discussion by Akornzzz** [ppy/osu#29868] [Ideas]
I took inspiration from the "daily challenge" and came up with the idea to introduce quests. Here's a rough idea of how they could work:

- Quests come in the form of "daily", "weekly" and "monthly".
- Quests can come in a few different rarities; "Standard", "Rare", "Unique"
- Some Quests can come in the form of; "SS a Map", "Get 5 A Ranks", "Pass 50 Maps", "Pass a (recommended diff + 0.5)* Map", "Pass 3 Maps by (Featured Artist)" and more.
- The difficulty of Quests would differ depending on how long they're available (hence "daily", "weekly" and "monthly")

That's the gist of how Quests could work, but what rewards would they give upon completion? Here's some ideas of what that could be.

- Quests give "quest points (QP)" which you can exchange for stuff like supporter, titles, medals and possibly more.
- Quests give titles (temporary)
- Quests give stats (like daily challenge profile stats)

That's everything I can think of at the moment for quest rewards. I personally really like the idea of Quest Points (QP) and having stuff to spend it on.

Overall, osu!Quests seems to me like it could be a fun feature that gives people something to do while working towards a reward.

**Comment by shinmorisawa** [comment]
hm... nice idea, but maybe not yet.

**Comment by Tenexxt** [comment]
I think a system similar to mappers guild (expect for actually playing the game and not mapping) could be used instead of making a new one.

The rewards for this could be the same as MG (Except with different badge design of course), with Monthly quests being a version of priority Quests, except i don't know what could be given instead of the keychain like in MG (too many players to make a keychain for)

The quest point system could also be the same _In My Opinion_, but I have no single idea who would menage this. This would have to be automated since you can't track every single player like with MG, because you have to rank a map to complete quests in MG which can be easily tracked and verified but with quests like this you could complete them on any map.
Makes it kinda hard to verify to menage because of that.
